paasta_tools/marathon_tools.py

```diff
@@ -1182,16 +1254,28 @@ def get_matching_appids(servicename: str, instance: str, client: MarathonClient)
     Useful for fuzzy matching if you think there are marathon
     apps running but you don't know the full instance id"""
     marathon_apps = get_all_marathon_apps(client)
-    return [app.id for app in get_matching_apps(servicename, instance, marathon_apps)]
+    return [app.id for app in marathon_apps if does_app_id_match(servicename, instance, app.id)]
```

Comment:
In here and the next 20+ lines it references "servicename". A while ago I changed all the serivce/servicename/service_name/instance/instancename, etc to just be service/instance, can you make these consistent?